azure-pipelines.yml

```diff
@@ -59,12 +59,31 @@ steps:
       Copy-Item -Path $sourcePath -Destination $(Build.ArtifactStagingDirectory) -ErrorAction Stop -Verbose -Force
   displayName: 'Copy package to ArtifactStagingDirectory'
 
+- pwsh: |
```

Comment:
Do we already have (and do we need?) the same code on the `dev` branch already?

Reply:
Yes, this code already exists in the dev branch -- this is how we build PowerShell 7 for V3. Without this porting this change to `v3.x/ps6`, we cannot push the package to the `AzureFunctionPreReleaseFeed`.